serverless.yml

```diff
@@ -0,0 +1,54 @@
+service: okta-hooks
```

Comment:
What does this do?

Reply:
That sets the name for all resources created in AWS.